modules/client-configuration/pages/clients-sleses.adoc

```diff
@@ -181,12 +181,9 @@ mount -o loop /tmp/<iso_filename> /srv/www/htdocs/pub/rhel
 
 When the image is mounted, you can synchronize the base media.
 To manually synchronize the channels, navigate to menu:Software[Manage > Channels].
-Click each channel in the list, and:
-
-. Navigate to the [guimenu]``Repositories`` tab.
-. Navigate to the [guimenu]``Sync`` subtab.
-. Click btn:[Sync Now] to begin synchronization immediately.
-. You can also create a synchronization schedule from this screen.
+Click each channel in the list, and navigate to menu:[Repositories > Sync].
```

Comment:
now it's running text.  intentionally? ;)
```suggestion
+
* Click each channel in the list, and navigate to menu:[Repositories > Sync].
```

Reply:
Yeah, it didn't seem worth making it into a procedure, especially since it's in a couple of different places.